soft4pes/model/grid/rl_grid.py

```diff
@@ -8,6 +8,9 @@
 class RLGrid:
     """
     Model of a grid with stiff voltage source and RL-load in alpha-beta frame. 
+    The states of the system are the grid currents in alpha-beta frame.
```

Comment:
The state of the system is the grid current in the alpha-beta frame.
The system input is the converter 3-phase switch position
The grid voltage is considered to be a disturbance

Reply:
Solved in next commit